### DISCUSSION TITLE
Get a frameless, always on top preview at a given position, like in the old picamera

### DISCUSSION THREAD

**Discussion by JamesRayn** [raspberrypi/picamera2#60] [Ideas]
Hi, it would be greate to get the availability to create a frameless, always on top preview at a given position, like in the old picamera.

In my case I simply added (after importing QtCore)

        self.qpicamera2.setWindowFlags(QtCore.Qt.FramelessWindowHint | QtCore.Qt.WindowStaysOnTopHint)
        self.qpicamera2.move(100, 100)

between L14 and L15.

https://github.com/raspberrypi/picamera2/blob/adaec856b9a23a2af2b34d7d1117c27e7c78d367/picamera2/previews/qt_gl_preview.py#L14-L15

May be you could add this to the public code as a optional parameter.

**Comment by ventilator** [comment]
This would be very useful for applications written in other UI frameworks (e.g. tk) which are using the camera preview, e.g. a photobox with fullscreen UI controls in the background beneath the camera preview
